src/time_util.rs

```diff
@@ -26,12 +28,37 @@ fn datetime_from_timestamp(context: &Timestamp) -> Option<DateTime<FixedOffset>>
 pub fn format_absolute_timestamp(timestamp: &Timestamp) -> String {
     static FORMAT_ITEMS: Lazy<Vec<chrono::format::Item>> =
```

Comment:
Perhaps rename this to something with DEFAULT in the name? If you follow the suggestion below, it could just be `PARSED_DEFAULT_FORMAT` or something similar.